app/components/join-section.hbs

```diff
@@ -34,8 +34,19 @@
       @route='index'
       class='join__link join__link--new'
       @disabled={{true}}
-    >Join the Squad</LinkTo>
+    >
+      Join the Squad
+    </LinkTo>
```

Comment:
Our linter was not liking the previous structure and messing up the code below. So, had to fix this along with it.